app/controllers/location_controller.rb

```diff
@@ -548,7 +548,7 @@ def create_location
                       action: :show,
                       id: @set_observation)
         elsif @set_species_list
-          redirect_to(controller: :species_list, action: :show_species_list,
+          redirect_to(controller: :species_lists, action: :show,
                       id: @set_species_list)
```

Comment:
I should have picked up on this earlier (in ProjectsController)
If trying to use helper paths when possible, then
```suggestion 
          redirect_to(species_list_path(@set_species_list)
```

Reply:
Thanks for catching these. I think i intentionally skipped them to avoid having extra conflicts to resolve on the upcoming Names/Locations. Double-checking now... yes they are corrected there.

Coming soon to a console near you! (I hope!)